### ISSUE TITLE
Unmet peer dependency in moment-timezone for core

### ISSUE THREAD

**Issue by espen** [fullcalendar/fullcalendar#7572]
### Reduced Test Case

https://fullcalendar.io

### Do you understand that if a reduced test case is not provided, we will intentionally delay triaging of your ticket?

- [X] I understand

### Which connector are you using (React/Angular/etc)?

No connector (vanilla JS)

### Bug Description

`@fullcalendar/moment-timezone@6.1.10" has unmet peer dependency "@fullcalendar/core@~6.1.10".`

My package.json includes @fullcalendar/moment-timezone, fullcalendar, fullcalendar-scheduler, moment and moment-timezone. I'm still on yarn v1, not sure if that could cause such issues.

I see there is https://github.com/fullcalendar/fullcalendar/issues/6839 which is similar and later fixed.

**Comment by acerix** [comment]
I tested in this example project and don't get that warning with yarn v1.22.21:

https://github.com/fullcalendar/fullcalendar-examples/tree/main/moment-timezone

Please post a [runnable, stripped-down demonstration](http://fullcalendar.io/wiki/Reduced-Test-Cases/) or steps to reproduce the bug.

**Comment by espen** [comment]
This is when using the fullcalendar bundle, not when specifying each individual package.

Am I doing something wrong here or should this work:

```
  "dependencies": {
    "@fullcalendar/moment-timezone": "^6.1.10",
    "fullcalendar": "^6.1.10",
    "moment": "^2.29.1",
    "moment-timezone": "^0.5.40"
  },
```

**Comment by espen** [comment]
Example package.json: https://github.com/espen/fullcalendar-examples/blob/package-error/moment-timezone/package.json

Should I do something different here @acerix or should this issue be reopened?

**Comment by acerix** [comment]
I don't believe `@fullcalendar` plugins are compatible with the bundled version, although it's not clear in the docs so I'll reopen.  

It should work with `@fullcalendar/core` as in this example:

https://github.com/fullcalendar/fullcalendar-examples/tree/main/moment-timezone

**Comment by espen** [comment]
Ok, would be great if this could work with the bundle due to version changes. If there is something new in the bundle I now have to manually add it.

Perhaps it is my limited experience with packages but I find it a bit confusing that I have to manually add @fullcalendar packages (adaptive, core etc) and also include the fullcalendar-scheduler which includes all the @fullcalendar packages to avoid this error.
